angularjs-portal-home/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -396,5 +396,17 @@ define(['angular', 'jquery'], function(angular, $) {
       };
 
       vm.init();
+  }])
+  .controller('OptionsController', [
+    '$scope', '$location', function($scope, $location) {
+      var expanded = 'expanded';
+      var compact = 'compact';
+      $scope.renderMe = false;
+      if (
+        $location.url().indexOf(expanded) > -1 ||
+        $location.url().indexOf(compact) > -1
```

Comment:
Is this a temporary or permanent fix? If permanent, any worries that someday there will be a "Mayflower Compact" app and weirdly the expand/contract widget is going to start showing on its details page? heh.

Reply:
I think "permanent" in the same sense that our current solution to toggling modes is "permanent." 

That said, it's probably worth looking for a better long-term solution for toggling expanded/compact modes, which would presumably change this implementation as well.

Reply:
yeah, we're just finding all the edges of customization that we made in uw-frame to support angularjs-portal. This is as permanent as we want it to be. This will be temporary as long as we make efforts to transform angularjs-portal into "just another frame app".

Reply:
actually, you're right @apetro, this would break if any entities have expanded or compact in their paths. this should really be a startsWith